hosts_manager.py

```
@@ -0,0 +1,81 @@
+    try:
+        with open(HOSTS_FILE, "r") as f:
+            lines = f.readlines()
+        
+        with open(HOSTS_FILE, "w") as f:
+            for line in lines:
+                if DOMAIN not in line:
+                    f.write(line)
+        
+        print(f"Removed {DOMAIN} from hosts file")
+        return True
+    except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)